scripts/createImage.sh

```diff
@@ -18,7 +18,8 @@ image_tag="$SPARK_VERSION-scala$SCALA_BIN_VERSION-java${JAVA_VERSION:-17}-ubuntu
 
 # There are no Docker images for Spark 3 and Scala 2.13, as well as for Spark 3.3.4 and any Scala
 if [[ "$SPARK_VERSION" == "3."* ]] && ( [[ "$SCALA_BIN_VERSION" == "2.13" ]] || [[ "$SPARK_VERSION" == "3.3.4" ]] ); then
-  image_prefix=spark
+  echo Checking for images for spark: $SPARK_VERSION scala: $SCALA_BIN_VERSION
+  image_prefix=spark-py
```

Comment:
This should be `spark` if `INCLUDE_PYTHON` is `false` and `spark-py` if `INCLUDE_PYTHON` is `true`.